ci-operator/step-registry/telcov10n/functional/cnf-ran/ibu-prune-argocd-apps/telcov10n-functional-cnf-ran-ibu-prune-argocd-apps-commands.sh

```
@@ -0,0 +1,27 @@
+set -e
+set -o pipefail
+
+if [ -f "${SHARED_DIR}/skip.txt" ]; then
+  echo "Detected skip.txt — skipping"
+  exit 0
+fi
+
+INVENTORY_PATH="/eco-ci-cd/inventories/ocp-deployment"
+
+mkdir -p ${INVENTORY_PATH}/group_vars
+cp ${SHARED_DIR}/all ${INVENTORY_PATH}/group_vars/all
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)